skiko/src/commonMain/kotlin/org/jetbrains/skia/PictureRecorder.kt

```diff
@@ -44,9 +45,10 @@ class PictureRecorder internal constructor(ptr: NativePointer) : Managed(ptr, _F
      *
      * @param bounds the cull rect used when recording this picture. Any drawing the falls outside
      * of this rect is undefined, and may be drawn or it may not.
+     * @param bbh optional acceleration structure
```

Comment:
It doesn't sound descriptive. Where it from?

Reply:
https://github.com/JetBrains/skia/blob/6726cd3e1ac4f30d6874c278008cdf24dfc9b470/include/core/SkPictureRecorder.h#L41C16-L41C27